.github/workflows/ci.yml

```diff
@@ -108,3 +111,65 @@ jobs:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
           CONTAINER_WORKSPACE: ${{ runner.workspace }}
         run: $CONTAINER_WORKSPACE/sonar/bin/sonar-scanner -Dsonar.host.url=https://sonarcloud.io -Dsonar.cfamily.threads=2
+
+  build-windows:
+    runs-on: windows-2019
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Move workspace
```

Comment:
maybe not needed for this project if path are not big as in the models compilation of Dynawo...

Reply:
Yes but I need deterministic path locations for dynawo-algorithms.cmd to setup correctly environment variables.

Reply:
not sure what you mean

Reply:
It's ok, i've found another way to do it.

Warning for the future: command line length for unit tests is close to maximum due to path and environment variables!